features/step_definitions/machine_set.rb

```diff
@@ -1,7 +1,14 @@
 Given(/^I pick a random machineset to scale$/) do
   ensure_admin_tagged
   machine_sets = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api")).
-    select { |ms| ms.available_replicas >= 1 }
+    select { |ms| ms.available_replicas >= 1 && ms.windows_machinesets != "Windows"}
+  cache_resources *machine_sets.shuffle
+end
+
+Given(/^I pick a random windows machineset to scale$/) do
```

Comment:
`Given(/^I pick a random (windows|linux) machineset to scale$/)`

Reply:
The original step can be updated slimilar to `I pick a random( windows)? machineset to scale` with condition to deal with windows machine scenario.